package/Dockerfile

```diff
@@ -1,34 +1,39 @@
-FROM nginx:1.21.6-alpine
+FROM nginx:1.24.0-alpine
+
+ENV DOCKER_VERSION=23.0.3
+ENV PLUGINS_VERSION=v1.2.0
```

Comment:
I guess this is a nit, but naming it more specifically allows us to put this automated bump in Renovate with a more descriptive variable (for example `CNI_PLUGINS_VERSION` instead of `PLUGINS_VERSION`)

Reply:
Good shout, amended it accordingly.